app/frontend/actions.js

```diff
@@ -864,6 +937,70 @@ module.exports = ({setState, getState}) => {
     validateSendFormAndCalculateFee()
   }
 
+  const updateStakePoolAddress = (state, e) => {
+    setState({
+      stakePools: state.stakePools.map((pool) => {
+        return Object.assign(
+          {},
+          pool,
+          pool.ID === parseInt(e.target.id, 10)
+            ? {
+              address: e.target.value,
+              ...fetchPoolInfo(e.target.value),
+            }
+            : {}
+        )
+      }),
+    })
+    validateDelegationAndCalculateDelegationFee()
+  }
+
+  const updateStakePoolPercent = (state, e) => {
```

Comment:
This is very obscure way of updating this value when it can be easily recomputed from the ground data. As I mentioned somewhere else, we actually don't need the `delegatedPercent` value at all

Reply:
Removed the `delegatedPercent` value.